<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_comp_2</class>
 <widget class="QDialog" name="fct_comp_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Liste des épreuves d'une catégorie (version 1, catégories fixées par interface)</string>
  </property>
  <property name="windowOpacity">
   <double>9.000000000000000</double>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Travail à réaliser : améliorer l'interface afin de proposer une liste des catégories possibles (saisies en dur dans l'interface).</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Saisissez une catégorie :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="text">
        <string>mixte</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>masculin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>feminin</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="table_fct_comp_2">
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>numEp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>nomEp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>formeEp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>nomDi</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>nbSportifsEp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>dateEp</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_fct_comp_2">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton_3</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_2</receiver>
   <slot>refreshResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>-60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_2</receiver>
   <slot>refreshResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>-84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_2</receiver>
   <slot>refreshResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>638</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>-80</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshResult()</slot>
 </slots>
</ui>
